cmd/package.go

```diff
@@ -100,7 +100,7 @@ func generateWindowsIcon(pngFilename string, iconfile string) error {
 	for _, size := range sizes {
 		rect := image.Rect(0, 0, int(size), int(size))
 		rawdata := image.NewRGBA(rect)
-		scale := draw.ApproxBiLinear
+		scale := draw.CatmullRom
```

Comment:
I noticed that scaling with NearestNeighbor was bad for small sizes (ApproxBiLinear uses   NearestNeighbor too).
CatmullRom is even better than BiLinear.

Sample comparison CatmullRom vs ApproxBiLinear:
![sample_icons](https://user-images.githubusercontent.com/1364496/82671269-dabdd200-9c3e-11ea-93ae-acd11b4d66ae.png)